### DISCUSSION TITLE
opendcs timezones

### DISCUSSION THREAD

**Discussion by marjgh** [opendcs/opendcs#178] [Q&A]
what is the difference between MST7MDT and MST time zones in OpenDcs, why don't we have just MST which is the standard time (UTC-7)? why MST7MDT is an option?

**Comment by MikeNeilson** [comment]
MST is moutain standard only. The offset from UTC will be 7 hours year
round.

MST7MDT accounts for daylight savings at the appropriate time. So the
offset will change to 6 at the appropriate time.

**Reply by marjgh** [reply]
Thanks for your quick response :)
Also i have another question. do we have any alternatives to Opendcs to do all the analysis, getting data, decoding, etc? in case in future the platform we use does not support it.

**Comment by MikeNeilson** [comment]
The USGS is working on something, however i believe that is is only doing the decoding portion.

However, since OpenDCS is written in Java I would be very likely a future platform wouldn't support it.

**Reply by marjgh** [reply]
I really appreciate your help. :)
